parlai/mturk/README.md

```diff
@@ -1,7 +1,167 @@
-`parlai/mturk` contains a few separate MTurk-related folders:
+The code in `parlai/mturk/` has been deprecated. All code for running crowdsourcing tasks can now be found in [`parlai/crowdsourcing/`](https://github.com/facebookresearch/ParlAI/tree/master/parlai/crowdsourcing), and all crowdsourcing tasks now utilize the [Mephisto](https://github.com/facebookresearch/Mephisto) platform.
 
-- **core**: contains all the files that are used to make the underlying MTurk interaction work properly
-- **tasks**: contains a number of tasks that have been launched on MTurk to collect various datasets (shared for reproducibility), as well as a few tasks that exist as examples of possible MTurk functionality
-- **scripts/**: contains various helpful review and cleanup scripts to manage tasks from the command line (especially after mistakes or major failures).
-- **webapp**: contains server and build files for the [Beta] ParlAI-MTurk frontend, which is currently under heavy development and may undergo many changes
-- **run_data/**: contains all of the cross-run state in sqlite databases (default `pmt_data.db` and `pmt_sbdata.db`), as well as any world data saved by the default behavior of `MTurkDataWorld` (in `live` and `sandbox` folders, following the behavior specified in `MTurkDataHandler`).
+Instructions on how to find all old `parlai/mturk/` code can be found in the sections below.
+
+### core/, scripts/, webapp/
+
+These folders can be restored by switching to the `final_mturk` tag of ParlAI:
+```bash
+git checkout final_mturk
+```
+
+
+### tasks/acute_eval/
+
+The current Mephisto-based version of the ACUTE-Eval task can be found in [`parlai.crowdsourcing.tasks.acute_eval`](https://github.com/facebookresearch/ParlAI/tree/master/parlai/crowdsourcing/tasks/acute_eval). The obsolete pre-Mephisto version of this task can be restored by switching to the `acute_eval` tag of ParlAI:
+
+```bash
+git checkout acute_eval
+```
+
+
+### tasks/convai2_model_eval/
+
+You can restore this task by switching to the `convai2archive`
+tag of ParlAI:
+
+```bash
+git checkout convai2archive
+```
+
+
+### tasks/dealnodeal/
+
+You can restore this task by switching to the
+[`mturk_archive`](https://github.com/facebookresearch/ParlAI/tree/mturk_archive)
+tag of ParlAI:
+
+```bash
+git checkout mturk_archive
+```
+
+If you just need to read the code, for reference, you may browse it
+[here](https://github.com/facebookresearch/ParlAI/tree/mturk_archive/parlai/mturk/tasks/dealnodeal).
+
+
+### tasks/image_chat/
+
+The current Mephisto-based version of the human+model image-chat task can be found in [`parlai.crowdsourcing.tasks.model_chat`](https://github.com/facebookresearch/ParlAI/tree/master/parlai/crowdsourcing/tasks/model_chat). The obsolete pre-Mephisto version of this task can be restored by switching to the `v0.10.0` tag of ParlAI:
+
+```bash
+git checkout v0.10.0
```

Comment:
i reckon we could make them all mturk_archive

Reply:
Hmm, that'd be hard - for instance, `convai2_model_eval` was removed *before* the `mturk_archive` tag, and I added a lot of new features to the pre-Mephisto turn annotations task a few days *after* the `mturk_archive` tag, so just switching to that tag wouldn't capture the final version of each of the tasks simultaneously